cmd/minishift/cmd/systemtray.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/anjannath/systray"
+	"github.com/minishift/minishift/pkg/minishift/systemtray"
+	"github.com/spf13/cobra"
+)
+
+// systemtrayCmd represents the systemtray command
+var systemtrayCmd = &cobra.Command{
```

Comment:
please create a folder for the `services` package, and make this file part of it.
This is not a command that needs to be called by users, unless debugging or power user I guess